examples/apps/cli/ftd.cmake

```diff
@@ -33,6 +33,8 @@ add_executable(ot-cli-ftd
 
 target_include_directories(ot-cli-ftd PRIVATE ${COMMON_INCLUDES})
 
+target_compile_definitions(ot-cli-ftd PRIVATE "OPENTHREAD_FTD=1")
```

Comment:
Should we do something like this instead (add all `OPENTHREAD_{XYZ}` configs)? 
Will make it similar to all other `ftd/mtd.cmake` files?
```
target_compile_definitions(ot-cli-ftd
    PRIVATE
        OPENTHREAD_FTD=1
        OPENTHREAD_MTD=0
        OPENTHREAD_RADIO=0
)
```